src/preact/base-element.js

```diff
@@ -131,6 +131,104 @@ export class PreactBaseElement extends BaseElement {
     return !this.usesLoading();
   }
 
+  /**
+   * Override to provide the Component definition.
+   *
+   * @protected {!PreactDef.FunctionalComponent}
```

Comment:
`!`

Reply:
There were so many to remove. You started a 🚋